packages/pancake-uikit/src/components/BalanceInput/BalanceInput.tsx

```diff
@@ -9,12 +9,20 @@ const BalanceInput: React.FC<BalanceInputProps> = ({
   onChange,
   currencyValue,
   inputProps,
+  inputMode = "numeric",
```

Comment:
No need for a parameter, it can be hardcoded inside the component directly

Reply:
I removed inputmode from params, change inputmode default to decimal and add min = "0" validation